prometheus-collectors/class-potential-multi-dataset-queries-collector.php

```diff
@@ -49,6 +50,21 @@ public function sql_query_log( $query ): void {
 					$blog_ids_count >= 3 ? '3+' : (string) $blog_ids_count,
 				]
 			);
+
+			$backtrace = function_exists( 'wp_debug_backtrace_summary' ) ? wp_debug_backtrace_summary( null, 4, false ) : []; // phpcs:ignore
+			\Automattic\VIP\Logstash\log2logstash(
+				[
+					'severity' => 'debug',
+					'feature'  => 'potential_multi_dataset_queries',
+					'message'  => 'Potential multi dataset query detected',
+					'blog_id'  => get_current_blog_id(),
```

Comment:
I believe `log2logstash` adds the `blog_id` for us?

Reply:
You are right. We do include a few default params.